scripts/protocgen2.sh

```diff
@@ -11,4 +11,5 @@ protoc_gen_install() {
 protoc_gen_install
 
 echo "Generating API module"
+(cd api; find . \! -name 'go.mod' \! -name 'go.sum' -delete; cd ..)
```

Comment:
should we rather delete files based on extension, i.e. `*.pulsar.go`, etc. what is there is a README and other stuff?

Reply:
makes sense - changed the script to only delete generated file extensions (.pulsar.go, .pb.go, .cosmos_orm.go, .pb.gw.go) and then delete all empty directories